inference-engine/src/cldnn_engine/CMakeLists.txt

```diff
@@ -12,6 +12,14 @@ if(CMAKE_COMPILER_IS_GNUCC)
     endif()
 endif()
 
+if(CLDNN_THREADING MATCHES "SEQ")
```

Comment:
Because cldnn plugin include engine that has CLDNN_THREADING in header. But I remove it and move task_arena_config from engine to ocl_engine to remove CLDNN_THREADING from engine.hpp.